[2024-01-01T09:33:35.063+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T09:33:30.152316+00:00 [queued]>
[2024-01-01T09:33:35.088+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T09:33:30.152316+00:00 [queued]>
[2024-01-01T09:33:35.089+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 3
[2024-01-01T09:33:35.191+0000] {taskinstance.py:2192} INFO - Executing <Task(HttpSensor): is_snow_api_ready> on 2024-01-01 09:33:30.152316+00:00
[2024-01-01T09:33:35.195+0000] {standard_task_runner.py:60} INFO - Started process 236 to run task
[2024-01-01T09:33:35.198+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'snow_dag', 'is_snow_api_ready', 'manual__2024-01-01T09:33:30.152316+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/servicenow_dag.py', '--cfg-path', '/tmp/tmpn6wld39m']
[2024-01-01T09:33:35.200+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask is_snow_api_ready
[2024-01-01T09:33:35.345+0000] {task_command.py:423} INFO - Running <TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T09:33:30.152316+00:00 [running]> on host cdceb6054911
[2024-01-01T09:33:35.676+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prady1900' AIRFLOW_CTX_DAG_ID='snow_dag' AIRFLOW_CTX_TASK_ID='is_snow_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T09:33:30.152316+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-01T09:33:30.152316+00:00'
[2024-01-01T09:33:35.677+0000] {http.py:130} INFO - Poking: api/now/table/incident
[2024-01-01T09:33:35.704+0000] {base.py:83} INFO - Using connection ID 'snow_api_id' for task execution.
[2024-01-01T09:33:37.748+0000] {base.py:293} INFO - Success criteria met. Exiting.
[2024-01-01T09:33:37.758+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=snow_dag, task_id=is_snow_api_ready, execution_date=20240101T093330, start_date=20240101T093335, end_date=20240101T093337
[2024-01-01T09:33:37.904+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-01T09:33:37.992+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
